### PR TITLE
EN: Some fixes

### PR DESCRIPTION
**Pull request by Frey8914**

### CODE REVIEW

File: Misc/clientorder_name.csv [hunk 1]
```diff
@@ -134,7 +134,7 @@ Notes_002#0,"""This order cannot be accepted<br>until $(0)."""
 Notes_003#0,"""This order can be completed<br>any number of times."""
 Notes_004#0,"""This order can be accepted until 23:59<br>JST."""
 Notes_005#0,"""This order has already been completed<br>and is no longer available."""
-Notes_006#0,"""This order is available until<br><utcdow>Wed 2:00</utcdow>."""
+Notes_006#0,"""This order is available until<br><utcdow>Wed 2:00</utcdow>"""
```
Comment (DabirSA) [inline Misc/clientorder_name.csv:137]: Why have you made this change?

Comment (DabirSA) [reply Misc/clientorder_name.csv:137]: image

File: UI/config.csv [hunk 2]
```diff
@@ -1167,10 +1167,10 @@ Config_085_Hint#0,"""Set whether to show the interface <br>(including windows) i
...
 Config_085_Hint_Switch#0,"""Set whether to show the interface <br>(including windows) in screenshots.<br>※Only applies to in-game functions."""
-Config_086_Name#0,"""Synchronized Physique Editing Settings"""
-Config_086_Choices_0#0,"""Synchronized Physique Editing"""
-Config_086_Choices_1#0,"""Separate Physique Editing"""
-Config_086_Hint#0,"""Set whether you want your Casts' human and robotic bodies to share the same proportions, or whether you want to edit them separately.<br><br>Humanoid-type and Cast bodies are synchronized by default. However, you can individually edit them by selecting \""Separate Physique Editing\""."""
+Config_086_Name#0,"""Body Shape Synchronization"""
+Config_086_Choices_0#0,"""Synchronized Body Shapes"""
+Config_086_Choices_1#0,"""Edit Body Shapes Individually"""
+Config_086_Hint#0,"""The Body Shapes of Body Types across race/gender<br>are synchronized by default. You can edit them<br>individually with \""Edit Body Shapes Individually\"".<br><br>Body Types will change depending on the worn<br>costume but Body Shape proportions will not.<br>With this setting, you can set Body Shapes<br>individually for each Body Type."""
```
Comment (SynthSy) [inline UI/config.csv]: This menu doesn't need manual line breaks unless you're splitting sentences to make a paragraph.

File: UI/ui_charamake_text_reb.csv [hunk 3]
```diff
@@ -99,18 +99,18 @@ Key_SetWearColor2#0,"""Color 2"""
 Key_RestoreDefaultColor#0,"""Reset to Default"""
 Title_CategoryMenu#0,"""Edit Menu"""
 Operation_CategoryMenu#0,"""Accept: <%Decide>\u3000Cancel: <%Cancel>\u3000Change Camera Distance： <swguidetype CameraLength>\u3000Rotate Camera： <swguidetype CameraTurn>\u3000Height Camera Movement: <swguidetype CameraUpDown>\u3000Toggle Costume: <%TabForward>\u3000Undo: <swguidetype Undo>"""
-Key_Category_Preset#0,"""Manage Created Data"""
-Help_Category_Preset#0,"""Save and Load your Data."""
+Key_Category_Preset#0,"""Manage Creation Data"""
```
Comment (SynthSy) [inline UI/ui_charamake_text_reb.csv:102]: Don't change this, or the thing below it.

File: UI/ui_equipnote.csv [hunk 4]
```diff
@@ -44,8 +44,8 @@ SD_Closable_Title#0,"""Collection File"""
 SD_Closable_Message#0,"""<pict $(0)>$(1)'s sheet has been received."""
 SD_Closable_Error_01#0,"""Could not redeem sheet."""
 Quest_None#0,"""None"""
-Quest_Level#0,"""Lv. $(0)+"""
-Quest_Level_New#0,"""Lv. $(0)+"""
+Quest_Level#0,"""Lv.$(0)+"""
```
Comment (SynthSy) [inline UI/ui_equipnote.csv]: Don't change this.

File: UI/quest_counter.csv [hunk 5]
```diff
@@ -286,9 +286,9 @@ CautionQuestPartyMemberMax#0,"""This quest cannot include $(0) or more<br>party
 CautionQuestPartyMemberJoinMax#0,"""Could not join party because <br>it would exceed $(0) members."""
 CautionQuestPlatform#0,"""Because not all of the Party Members<br>meet the following order conditions, you cannot<br>begin the quest:<br><br>・$(0)"""
 CautionQuestRaidQuest#0,"""Because this does not meet the quest conditions,<br>you cannot accept the orders."""
-EnemyLevel#0,"""Lv. $(0)+"""
-EnemyLevelRange#0,"""Lv. $(1)～$(0)"""
-EnemyLevelUnknown#0,"""Lv. ???"""
+EnemyLevel#0,"""Lv.$(0)+"""
```
Comment (SynthSy) [inline UI/quest_counter.csv]: Don't change this.

File: UI/quest_counter.csv [hunk 6]
```diff
@@ -521,9 +521,9 @@ CautionQuestRaidQuest#1,"""Because it does not meet the quest conditions,<br>the
 ReasonPartyLevel#2,"""<pict mainclass_icon>Lv.$(0)～"""
 ReasonPartyMax#1,"""・Party count $(0)"""
 ReasonEventFlag#3,"""・Conditions/Client Orders"""
-EnemyLevel#1,"""Lv. $(0)+"""
+EnemyLevel#1,"""Lv.$(0)+"""
```
Comment (SynthSy) [inline UI/quest_counter.csv]: Don't change this.

File: UI/config.csv [hunk 7]
```diff
@@ -1326,7 +1326,7 @@ Config_143_Choices_3#0,"""90fps"""
...
 Config_143_Choices_5#0,"""144fps"""
 Config_143_Choices_6#0,"""165fps"""
-Config_144_Name#0,"""Body Type Synchronization"""
-Config_144_Choices_0#0,"""Synchronize Body Type"""
-Config_144_Choices_1#0,"""Edit Body Types Individually"""
-Config_144_Hint#0,"""Body types in NGS & PSO2 are automatically synchronized,<br>but if you want to edit them individually, please select<br>\""Edit Body Types Individually\"". (Only available within the Salon)"""
+Config_144_Name#0,"""NGS/PSO2 Body Shape Synchronization"""
+Config_144_Choices_0#0,"""Synchronized Body Shapes"""
+Config_144_Choices_1#0,"""Edit Body Shapes Individually"""
+Config_144_Hint#0,"""NGS/PSO2 Body Shapes are automatically synchronized, but if you want to edit them individually, please select \"Edit Body Shapes Individually\". (Only available within the Salon)"""
```
Comment (alama) [inline UI/config.csv]: Why did you remove the escape for quotes?

File: Orders/_Bingo/bo00001374.csv [hunk 8]
```diff
@@ -1,2 +1,3 @@
 Title#0,"""Bingo Order 99"""
+Explanation#0,"""[Enemy Defeat]<br>Dio Hunar<br><br>[Target Level]<br>Lv.41+"""
```
Comment (DabirSA) [inline Orders/_Bingo/bo00001374.csv]: You've got an extra line here somehow

File: Orders/_Bingo/bo00001520.csv [hunk 9]
```diff
@@ -1,2 +1,3 @@
 Title#0,"""Bingo Order 99"""
+Explanation#0,"""[Enemy Defeat]<br>Dio Hunar<br><br>[Target Level]<br>Lv.41+"""
```
Comment (DabirSA) [inline Orders/_Bingo/bo00001520.csv]: Extra line here too

**Comment by DabirSA** [comment]
Could you make sure that Inventory is capitalised everywhere? Once that's done I think we should be good.

**Comment by DabirSA** [comment]
Looks good to me.
